src/__tests__/App.test.tsx

```diff
@@ -1,12 +1,14 @@
 import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
 
 import App from '../components/App';
 import { render } from './utils/setupTests';
 import { screen } from './utils/test-utils';
 
 describe('App', () => {
-  test('Should render search view on default route', () => {
+  test('Should render search view on default route', async () => {
     render(<App />);
+    await act(async () => void jest.runOnlyPendingTimers());
```

Comment:
this is sometime required to avoid `an update to ___ was not wrapped in act()` warnings